### ISSUE TITLE
Change storing data strategy

### ISSUE THREAD

**Issue by WTCa100** [WTCa100/PIrun---The-Final-Math-Game#4]
# Introduction
Currently `Player` and its `Pset` info is being stored in 4 different files.
- Highscores
- Scoreboard
- Details
- Pset (directory with a different files name after player `id`)
Thus loading files such as `Scoreboard` `Details` and `Highscores` is really similar. I would like to remove `Scoreboard.csv` file so that scoreboard info would get loaded via `Details.csv` which I find more convenient. No matter what stats player did and how he did it we have the same information inputted and outputted in both `scores` and `details` except that details are enriched in a few extra information.
### Scoreboards info extraction
```C++
	while (std::getline(ssRow, rawData, cDelimeter))
	{
		switch (nCell)
		{
		case 1: *tmpId = std::stoi(rawData); break;
		case 2: *tmpUserName = rawData; break;
		case 3: *tmpPoints = std::stod(rawData); break;
		}
		if (nCell == 3)
		{
			Player* tmpP = new Player(*tmpId, *tmpUserName, *tmpPoints);
			mapOut.insert(std::make_pair(tmpP->givePlayerId(), *tmpP));
			delete tmpP;
			delete tmpId; delete tmpUserName; delete tmpPoints;
			nCell = 1;
		}
		else nCell++;
	}
```
### Details info extraction
```C++
	while (std::getline(loadDetails, strLine))
	{
		switch (nLine)
		{
		case 1:
			*tmpId = std::stoi(strLine);
			break;
		case 2:
			*tmpPoints = std::stod(strLine);
			break;
		case 3:
			*tmpComp1 = std::stod(strLine);
			break;
		case 4:
			*tmpAct = strLine[0];
			break;
		case 5:
			*tmpComp2 = std::stod(strLine);
			break;
		case 6:
			*tmpExp = std::stod(strLine);
			break;
		case 7:
			*tmpActual = std::stod(strLine);
			break;
		case 8:
			*tmpCor = std::stoi(strLine); // Value in this line will always be either 1 or 0
			break;
		}
		if (nLine == 8)
		{
			Problem* tmpProb = new Problem(*tmpId, *tmpPoints, *tmpComp1, *tmpAct, *tmpComp2, *tmpExp, *tmpActual, *tmpCor);
			PSearchFor.AssignProblem(*tmpProb);
			delete tmpProb;
			nLine = 1;
		}
		else nLine++;
	}
```

## Conclusion and further steps
In conclusion, the same results would be given if we decided to just operate on `Details.csv`, but the program would have to do more loading and it would have to use more dynamic memory, but I think that it is worth making such a change.
### TODO
- Remove scoreboard
- Rename `Details.csv` into `PlayersData.csv`
- Operate only on just renamed `PlayersData.csv`

**Comment by WTCa100** [comment]
As of 04.11.2022 game only handles one file `PlayersData.csv` - issue is being closed
